frontend/package.json

```diff
@@ -3,10 +3,10 @@
     "version": "0.1.0",
     "private": true,
     "scripts": {
-        "dev": "vite --host localhost --port 5005",
-        "dev2": "vite --host --port 5005",
+        "dev": "vite --host localhost --port 5000",
```

Comment:
I changed the port because I had a conflict with another process running on my computer. ^^

Reply:
Oh I see, I'll revert this